### ISSUE TITLE
Function that automatically downloads files from zenodo

### ISSUE THREAD

**Issue by niekdejonge** [iomega/ms2query#77]
The average user of MS2Query does not want to make library files by itself, therefore it might be a good idea to add a function that downloads all needed files from zenodo (sqlite, embeddings, models).

Once this is implemented, we can change the readme file of MS2Query, to explain how MS2Query can be run with these default settings.

**Comment by justinjjvanderhooft** [comment]
Good point! We should add a warning/message about the space it needs on your local disk or server!

**Comment by niekdejonge** [comment]
For the readme file see also #39
